tests/test_utils.py

```diff
@@ -0,0 +1,94 @@
+import contextlib
+import logging
+import sys
+from pathlib import Path
+
+import pytest
+
+from remarx.utils import configure_logging
+
+
+@pytest.fixture(autouse=True)
+def reset_logging():
+    """Ensure a clean logging configuration for each test."""
+    root_logger = logging.getLogger()
+    for handler in root_logger.handlers[:]:
+        root_logger.removeHandler(handler)
+        with contextlib.suppress(Exception):
+            handler.close()
+
+
+def test_configure_logging_default_creates_timestamped_file(tmp_path, monkeypatch):
+    """Test that the default configuration creates a timestamped log file."""
+    # Run in a temporary CWD so logs land under tmp_path/logs/
+    monkeypatch.chdir(tmp_path)
+    created_path = configure_logging()
+
+    assert isinstance(created_path, Path)
+    assert created_path.parent.name == "logs"
+
+    # Check that the log file name starts with "remarx_" and ends with ".log"
+    assert created_path.name.startswith("remarx_")
+    assert created_path.suffix == ".log"
+    assert created_path.exists()
+
+    # Stanza logger level default
+    assert logging.getLogger("stanza").getEffectiveLevel() == logging.ERROR
```

Comment:
Can you do the same thing for a different context to check that it is the expected level? 

Or can you use logic similar to what you're doing in the `reset_logging` fixture to inspect the configured logging handler?